modules/fedimint-ln/src/lib.rs

```diff
@@ -125,6 +125,7 @@ pub struct LightningGateway {
     pub mint_pub_key: secp256k1::XOnlyPublicKey,
     pub node_pub_key: secp256k1::PublicKey,
     pub api: String,
+    pub active: bool,
```

Comment:
One thing that's weird about this is that now gateways will be saved with this attribute in the database. But if it's in the database, then it's `active` by definition at this point. I'm not sure if there's something better to do ...

Reply:
Maybe to prepare for later changes discussed above, when there would be multiple gateways in client DB. But you're right that should be introduced in the future, I think we can just remove that attribute for now and change the logic to implicitly view the gateway as active (what it kind of already does.)

Reply:
But then you should you display `active: true` in the CLI output? Maybe just json-serialize it with serde_json and manually add?

Reply:
Dumb question if the client has more gateways in DB, still only one will ever be active right ? In that case just don't bother putting this in the json and just e.g println `Found active gateway\n`, `pretty_print(json)`

Reply:
Hm, If having a gateway reference in the client DB implies that it's active then the idea to append the active attribute whenever we read from the DB makes sense. The only challenge I see is that after we append the active flag to a deserialized gateway, I think we lose typing / would need to define a new type which we can 'transport' downstream, to the cli or any other client.

Reply:
Another weirdness with having the active field in Gateway definition is that gateways might be instantiated with `active: true` but this does not reflect the perspective of the clients who are observing the gateway. Only clients should be able to set the flag on Gateway references, yes?

Reply:
^^ yes. I don't think `active` really makes sense to store in the database.

In the `main.rs` file where we handle the CLI commands you could probably call `serde_json::to_value(gateway)` and then just update the result to have a `key: <active>` field. This way we only change the CLI code and don't change the database schema.